sequencing/ATACSeq/jobSubmission/6_batchRunGenotypeCheck.sh

```diff
@@ -0,0 +1,157 @@
+## INPUTS:                                                                                                                     || 
+## --array -> Number of jobs to run. Will select sample(s) corresponding to the number(s) input                                ||
+## $1 -> <project directory> Directory of the project, location of the config file                                             ||
+## $2 -> <STEP> Specify step to run: COMPARE, GENCHECK. Can be combined. Default is to run all                                 ||
+##                                                                                                                             ||
+## OUTPUTS:                                                                                                                    ||
+## - Potential contaminated samples will be specified in: ${META_DIR}/potentialSwitches.txt                                    ||
+##                                                                                                                             ||
+## REQUIRES:                                                                                                                   ||
+## - File with samples IDs and their matching VCF IDs: ${META_DIR}/matchedVCFIDs.txt                                           ||
+## - Config.txt file in <project directory>.                                                                                   ||
+## - The following variables specified in config file: META_DIR, MAIN_DIR, LOG_DIR, ALIGNED_DIR, SCRIPTS_DIR, PROJECT,PEAK_DIR ||
+## - Version/directory of the following modules should be specified in config file: PICARDVERS, RVERS, SAMTVERS                ||
+## - Softwares: Picard, GATK, SAMtools, R, Pandoc,                                                                             ||
+## - For modules or references required, please refer to each subscript run in this script.                                    ||
+## - Subscripts to be in ${SUB_SCRIPTS_DIR} = ./subscripts                                                                     ||
+## - Subscripts: compareBamWithGenotypes.sh, searchBestGenoMatch.sh                                                            ||
+## - R subscripts to be in ${RSCRIPTS_DIR} = ./Rscripts                                                                        ||
+## - R scripts: collateSampleChecks.Rmd                                                                                        ||
+##                                                                                                                             ||
+## ============================================================================================================================##
+
+## ============ ##
+##    SET UP    ##
+## ============ ##
+
+## print start date and time
+echo Job started on:
+date -u
+
+## load config file provided on command line related to the specified project
+source "${1}/config.txt"
```

Comment:
In case the config file is not found (because the user typed it incorrectly or gave the wrong path *etc.*), could you put a error message here?

So you could do:
```bash
source "${1}/config.txt" || { echo "Could not find config.txt at $1"; exit 1; }
```

Or something to that effect.

Reply:
Yes, I will add that check